libarchive/archive_util.c

```diff
@@ -633,70 +633,22 @@ __archive_ensure_cloexec_flag(int fd)
  * Utility function to sort a group of strings using quicksort.
  */
 static int
```

Comment:
Really, libarchive shouldn't be vending this at all.  Please add `#if ARCHIVE_VERSION_NUMBER < 4000000` around this function (and the next) to help make that happen.

Reply:
Great!  Thank you!